mobile/src/libs/components/button/button.tsx

```diff
@@ -6,8 +6,9 @@ import { Text } from '#libs/components/components';
 import { styles } from './styles';
 
 type Properties = {
-  label: string;
+  label?: string;
   onPress: () => void;
+  iconSourceSvg?: JSX.Element;
```

Comment:
I think `ReactNode` is more priority type, if there'll be no errors. Also it's better to name it like `iconSource` or `icon` without extra type

Reply:
@Dmytro-Kucherenko changed to ReactNode, but what about naming, i think this one that i use is more descriptive, to understand that its svg, because we are using different types here in code